FakeItEasy.sln

```diff
@@ -69,6 +69,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "docs", "docs", "{072853A5-3
 		docs\assigning-out-and-ref-parameters.md = docs\assigning-out-and-ref-parameters.md
 		docs\bootstrapper.md = docs\bootstrapper.md
 		docs\calling-base-methods.md = docs\calling-base-methods.md
+		docs\calling-wrapped-methods.md = docs\calling-wrapped-methods.md

```

Comment:
Ha. Oops. Nice catch.

Reply:
I couldn't find source-stepping.md in the solution, so I had to add it, and looked for other missing ones while I was at it.